### DISCUSSION TITLE
Change From Email Address to noreply@whatever.org

### DISCUSSION THREAD

**Discussion by gismcieri** [ecederstrand/exchangelib#957] [Q&A]
Hi,

Is it possible to send the email from an account but have the reply to address be something like noreply@whatever.org.  I am hoping this is something that can be done.  The noreply address is an unattended mailbox and doesn't have send permissions.

Thanks,
Matt

**Comment by ecederstrand** [comment]
I think this is a duplicate of https://github.com/ecederstrand/exchangelib/issues/544?
